Assets/__Scripts/MapEditor/AudioTimeSyncController.cs

```diff
@@ -247,6 +259,10 @@ public void RefreshGridSnapping()
     public void OnTogglePlaying(InputAction.CallbackContext context)
     {
         if (context.performed) TogglePlaying();
+
+        // if play is held and released a significant time later, cancel playing instead of merely toggling
+        const float cancelInputDuration = 0.3f;
```

Comment:
is there a reason why this constant is defined in the method? wouldn't hurt to move it out of the method

Reply:
Just personal preference, moved it up next to `playStartTime`.